dquartic/model/model.py

```diff
@@ -371,7 +376,7 @@ def train_step(self, x_0, ms2_cond=None, ms1_cond=None, noise=None, ms1_loss_wei
                     )
         elif self.pred_type == "x0":
             # Predict x0
-            x0_pred = self.model(x_t, t, ms2_cond, ms1_cond)
+            x0_pred = self.model(x_t, t, ms2_cond, ms1_cond, precursor_cond)
```

Comment:
Same comment as previous above, relating to the current unet1d models expected inputs